<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWindow</class>
 <widget class="QMainWindow" name="ResultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#F9F6F4 ;
color:#1B224D; 
font-family: 'Roboto', sans-serif;
font-size: 15px;
font-weight: bold;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1021</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="deviceTable">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StepMode">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>450</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #F9F6F4;
 background-color:#1B224D; 
 font-size: 16px;
 border-radius: 0.5em;
 width: 12em;
 height: 3em;
 text-transform: uppercase;
 font-weight: bold;
  font-family: 'Roboto', sans-serif;
 text-align: center;
margin: 3px;</string>
    </property>
    <property name="text">
     <string>Step Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
